pkg/common/v1alpha2/common.go

```diff
@@ -29,3 +37,27 @@ func GetManagerAddr() string {
 		return KatibManagerService + "." + ns + ":" + KatibManagerPort
 	}
 }
+
+func getKatibClientAndConnection() (*katibClientAndConnection, error) {
+	addr := GetManagerAddr()
+	conn, err := grpc.Dial(addr, grpc.WithInsecure())
+	if err != nil {
+		return nil, err
+	}
+	kc := &katibClientAndConnection {
+		Conn: conn,
+		KatibClient: api_pb.NewManagerClient(conn),
+	}
+	return kc, nil
+}
+
+func RegisterExperiment(request *api_pb.RegisterExperimentRequest) (*api_pb.RegisterExperimentReply, error) {
```

Comment:
Does it have to be put in api_util itself? I feel that We should keep grpc api in one place and It doesn't fit to be in common.

Reply:
I think we should put it into common part since both experiment and trial, or even suggestion service will call it to access DB by katib-manager grpc

Reply:
When would `trial` need to call `RegisterExperiment`?

Reply:
I mean I will also add `RegisterTrial` into this file, both `RegisterTrial`  and `RegisterExperiment` need katib-client to call katib-manager to visit DB.